src/main/groovy/com/bmuschko/gradle/docker/tasks/container/DockerWaitContainer.groovy

```diff
@@ -19,6 +19,8 @@ class DockerWaitContainer extends DockerExistingContainer {
     @Override
     void runRemoteCommand(dockerClient) {
         logger.quiet "Waiting for container with ID '${getContainerId()}'."
-        dockerClient.waitContainerCmd(getContainerId()).exec()
+        def containerCommand = dockerClient.waitContainerCmd(getContainerId())
+        ext.exitCode = containerCommand.exec()
```

Comment:
This should be set to a property within the task itself (e.g. public int exitCode).